code/src/nuvla/connector/nuvlabox.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, **kwargs):
         self.nuvlabox_api.headers = {'Content-Type': 'application/json',
                                      'Accept': 'application/json'}
 
-        self.nuvlabox_id = kwargs.get("nuvlabox_id")
+        self.nuvlabox_id = self.job.nuvlaedge_id
```

Comment:
The `target-resource` is not always the NuvlaEdge so this change will break some jobs (for example the `nuvlabox_log_fetch`).
Also, all the codes instantiating the NuvlaBox class are still passing the kwarg `nuvlabox_id` to the constructor but it is then ignored.

Reply:
Looking at the resource log fetch, I see your point.
We have a mismatch of ways...

Reply:
OK... looking closer this comes from the re-factoring and merge with main...
I will change "back" to the previous method an re-test to make sure my code does not break.